book_source/source/module-7-geospatial-data/lesson-20-dissolve-join/hurricane-florence/HurricanFlorence.ipynb

```diff
@@ -1 +1 @@
z113eRcAvO5ZZ+LCJz4c7XZ73SzaoijKNu+O4yAIArRarQ0LZK+mlmWrc0wQBANLvviaJCIionnDwM+YqA4uNF27mSWSZRmazeZY9qXrOoIgGNv+Zpnv+3tm4qUmqrNW4NlxHJimWU6+p2mvHCtbmaeH4arDD+Lnzj99qG1VEEdl+OR5jjzPy+LYKpCsXmuqeLZpmvB9f01mDo8nIiIimkcM/IxRt9ud9hCIaIaoFuizFPQBeoGfdrtdiWD3XlgSOIx5eRhOPcHF37/iKegm2VA181zXRafTgWmayPN8w4LjpmnCsiyYpllexiAPERER7RUM/IyBlBJhGFZ+8sYPudXFyevepAIntVoNSZKUx0H/8bDRz8pGRWlXX7ZRzRJV12Sjfahzm6ZpA+2tqxD0oR7X72W7bFBneKacvr+3XDTutDbdzvd9mKYJIQTyPC+PUVVvzDCMMptOdfNafawTERER7RUM/OyQ+ra+Cm2NaXZxMrI3ZVmGKIrgui50XV83eDyuAt7rFbPtv7z/f/XPNE3U63W02224rgtN0xBFEc93FeK6Lhyr91Zu6hrCtNpfQGzl5vtaiNIcblBH2F5edxtd12FZFoDe8dpoNNYcx1sFNImIiIj2EgZ+xiBJkmkPgYhmVBzHcF0XUkosL68/0Z0WFZSq1WooimLTpTQ0HWEYwrRs6JqAMQedpO5pRnjc730ON/3xD8MP6uhsEPxZncGz0y5cRERERPNs6E+JQghdCPF1IcRnVn4/QQjxBSHELSv/75vcMKtLCDFQM4CIaBRSSnQ6HWiaVmYxVInqfKRpGoPcFaWvrPEqMNvZPv1e8+FrcX8nge3V1vwtz3M0m00sLS2VhZyJiIiIaGOjfD34OgA39v3+mwAukVKeCeCSld/3pP60cyKiUaVpiiiK4Ps+XNed9nDWUK2vGfipHs/zyp9nrKP7pi67+Rhe8O6v4O5mBMMN1vy9f0kis3yIiIiINjdU4EcIcQqA5wH4u76LXwDggys/fxDAC8c7tNkhhIDneajVavwASkQjUwXiu90uHMeZ9nDWUAV0q2a3zreqDXgV5ULH6z72dWTzFPVZcXLDwZEHu6g5vdbrqwkhYBhcsU5ERES0lWE/Mb0TwOsB9OdcnySlvAcApJT3CCEeOu7BzRIhBHRdR61WQ6vVquwkgajq1GQuCNZ+yz/vVCCj0WhgeXm5MueRLMtgWdZQ7bVp9+i6Ds8y8J833It3vORszMvXDg9vOPi9C/8fnP+YhwCQaLfb6x57qiC6Nge1jYiIiIgmacvAjxDiQgBHpZTXCiGeOeoNCCFeDeDVAHDaaaeNPMBZolrHBkGAVmvzVrREtD7VKW+vtgvPsgymaWJhYQFxHEPXdURRNNWgSxzHlQ3EVSU4Ng2u5+NYK0a0qpNX95YrER4+BPfAQXhnnjul0Q3vYXUHYZpD1wQe97Aa3nDRWXjUiR7arc2LnauW7URERES0uWEyfs4DcJEQ4kcAOADqQogPA7hPCHHySrbPyQCOrndlKeV7AbwXAM4555y5/4SuMn88z0O32532cAbs5QkSzZY8zxFF0bSHMTVhGMJxHJimWWY/LS4uTm08apmX67plsee9pmrnT13XoWsa/uQ/jpfeMzQN8W1X4v5PvQ0yi9G5/mKc9MLXw+8P/ggByN5/YogcIYnB+13Ilfo6ANZ7SFQcZr09l523Vn5/9EN8/P0rnoL9gQ1NA3Qh0IozuKaGKNz6/VPTNAZ+iIiIiIawZeBHSvlbAH4LAFYyfn5NSvmTQoi3A3gFgLes/P/JCY5zpgghYFkWwjCs3GSBqOr4mumJoghZlqFW662wnWadGSklut0uPM9DmqZ7NhurSmzbhqYJfOa6ewAAS90U1/zeBfilX/wk3p3FAACZxXjxSQ/iL9/0I5AAtJUgSVH0h3PWO6aOB1OSrEAhJRxT7/1F9P66VcBFBYf69yb7bk6i140sSTNkaQzb83pBxThCGG99/9X7LBERERFtbSdVEd8C4ONCiFcBOALgJeMZ0nyQUsIwDNbEIKJtUctG4ziuRPag6ugVBAGSJKnEmIBqF16eFNfzoBkm/unqI0hWijqf86aLAQDde06AMGzILIYwbHz8nhPwmd/6LADANjTc/MbnotlcWrPPffv2ATj+PKugyn/ftYRLbzyK3/yRxwMA0rxAe7m57bFrmgbTNJGmKYIgQJYmcF0XWZaNlOVX1aWHRERERFU0UuBHSnkZgMtWfn4AwLPGP6T5sBcnI0Q0XlU7jyRJgqIo4Pt+JZez7gV+UMNimOGVf/sV3HD32ho43pnnYv9Fr1+3xo9n6QPbuq4L6CZ0TeDBToL/uuUYnnTqAk49odci/r2Xfwfv+PzNiLMCX7jxPlzyq8+EqWs7WvLXaDQAAN1ud+DY1nV9o6usEQQBdF3nMi8iIiKiIbEP6gRxOQQRbVdRFGi32/B9H5ZlodncfpbFOGVZVo6r0Wig2+1OJbNxr9R3cT0PhmFCSiCXEjff18LPfuha3Lu8cXaMd+a56xZ1NnStbPvuBzVYpoFPfeMupLnEb3/ivxFnx4tE/+qzH4Nf/MEz8fyzT8YvfvTruOaORTzmd/4D337Tc2Ga1siBnyAIBlqv53leduRK0xSmaaJWq6HT6aAo1m9NL4SA67os6kxEREQ0IgZ+JqhK39ZXaSyTwokAzZs0TRGGITzPg2EYlQkm53mO5eVlWJYF3/cRRdGuF+P2fb8MHkzaNM4tuq5Ds1w4tonXfPhaPP30E/Hp6+7GN767hDTf3rncNjQYugY3qCMtJF7wF5fjxnvW70D54oOnoB1l+MShO/GhVz0Vn7n+HvzRp7+FpW6CwDawb98+JGmGTnvzDpaa1ssQMk0TQC+gqepXWZYFTdPQ6XTKwFCj0UAURUiSpCwqDgCmacL3fQA81xMRERGNioGfCbJte093JiLaLk7sjlMt3Wu1WjlpjuMhqt/ugiRJIKWE53m7fq5TmSK7RS1tM01z
```

Comment:
Do we use `data.zip` in this slide? Same for Lesson 19 Hurricane Florence. Also, Lesson 19 is missing the stormhistory.csv from the data files download